vdb/extensions/__init__.py

```diff
@@ -57,8 +60,15 @@ def loadExtensions(vdb, trace):
                 module.vdb = vdb
                 module.__file__ = modpath
                 spec.loader.exec_module(module)
+
+                if not hasattr(module, 'vdbExtension'):
+                    logger.info("Skipping python module %r, not a valid extension", modpath)
```

Comment:
Could we make it a bit more explicit like `module has no vdbExtension() function` here and below? We've found a bit of spoon feeding in log messages goes a long ways :)

Reply:
```suggestion
                    logger.warning("Skipping python module %r, not a valid extension (module has no vdbExtension() function)", modpath)
```